src/covid19sim/hydra-configs/run-type/collect-all.yaml

```diff
@@ -0,0 +1,3 @@
+COLLECT_LOGS: True
```

Comment:
If there are two files in this folder, what is the default file that will be loaded?